README.md

```diff
@@ -163,3 +163,20 @@ Note that if you already have an instance running, this will likely throw an err
 The application's default log in is admin / change this password now
 
 Note the subtle nudge ;)
+
+## System Requirements
+
+- For big files with a small database footprint (such as video, audio and images) you'll need more HD space than CPU or RAM
+- For text documents you should consider some decent RAM as ElasticSearch is pretty greedy on memory for full text search
+
+The bare minimum you need to be able to run Uwazi without bottlenecks is:
+
+- 4 GB of RAM (reserve 2 for Elastic and 2 for everything else)
+- 2 CPU cores
+- 20 GB of disk space
+
+For development:
+
+- 8GB of RAM (depending on whether the services are running)
+- 4 CPU cores
+- 40 GB of disk space
```

Comment:
For development purposes I think 20 GB (same as for production) should be enough, as developers probably don't need a huge dataset to develop with

Reply:
Updated